README.md

```diff
@@ -0,0 +1,83 @@
+# Baltimore County Javascript Utilities
+
+A group of javascript utilities that have been found to be useful across different projects at Baltimore County.
+
+## Config Utilities
+
+The config utilities help us share configuration values based on the cms environment they are in. Our current cms makes it extremely difficult to manage our config values in our apps.
+
+### SetConfig
+
+The constructor takes one parameter "values" which an object which contains configuration for the following environments: `local, development, staging, production`;
+
+
+**Usage**
+
+```js
+import { SetConfig } from '@baltimorecounty/javascript-utilities';
+
+const configValues = {
+	local: {
+		apiRoot: 'http://localhost:1919/api'
```

Comment:
We all open on port 3000 with our react apps but since this can technically be used with non react apps I guess that doesn't matter. But in the interest of making things consistent we should probably have a standard for what ports to run our apps locally.

Reply:
apiRoot, is actually just a sample key. The logic for detecting if something is using localhost is

```js
hostname.indexOf('localhost') > -1
```

or if there are no other matches it will default to local. However, you will get warned when a environment or key doesn't exist.

Reply:
added another key to the docs, to make this less confusing.

Reply:
Yeah that makes sense. And like I said since this really isn't react specific it doesn't really matter what port we list.